docs/source/common/lightning_cli.rst

```diff
@@ -757,6 +757,34 @@ Instantiation links are used to automatically determine the order of instantiati
     <https://jsonargparse.readthedocs.io/en/stable/#jsonargparse.core.ArgumentParser.link_arguments>`_.
 
 
+Variable interpolation
+^^^^^^^^^^^^^^^^^^^^^^
+
+The linking of arguments is intended for things which are meant to be non-configurable. This improves the CLI user
+experience since it avoids the need for providing more parameters. A related concept is
+variable interpolation which in contrast keeps things being configurable.
+
+The yaml standard defines anchors and aliases which is a way reuse content in multiple places of the yaml. This is
+supported in the ``LightningCLI`` though it has limitations. Support for OmegaConf's more powerful `variable
+interpolation <https://omegaconf.readthedocs.io/en/2.1_branch/usage.html#variable-interpolation>`__ will be available
+out of the box if this package is installed. To install it run :code:`pip install omegaconf`. Then to enable the use
+of OmegaConf in a ``LightningCLI``, when instantiating a parameter needs to be given for the parser as follows:
+
+.. testcode::
+
+    cli = LightningCLI(MyModel, parser_kwargs={"parser_mode": "omegaconf"})
+
+With the encoder-decoder example model above a possible yaml that uses variable interpolation could be the following:
```

Comment:
```suggestion
The linking of arguments is intended for things that are meant to be non-configurable. This improves the CLI user
experience since it avoids the need for providing more parameters. A related concept is
variable interpolation which in contrast keeps things being configurable.

The YAML standard defines anchors and aliases which is a way to reuse the content in multiple places of the YAML. This is
supported in the ``LightningCLI`` though it has limitations. Support for OmegaConf's more powerful `variable
interpolation <https://omegaconf.readthedocs.io/en/2.1_branch/usage.html#variable-interpolation>`__ will be available
out of the box if this package is installed. To install it run :code:`pip install omegaconf`. Then to enable the use
of OmegaConf in a ``LightningCLI``, when instantiating a parameter needs to be given for the parser as follows:

.. testcode::

    cli = LightningCLI(MyModel, parser_kwargs={"parser_mode": "omegaconf"})

With the encoder-decoder example model above a possible YAML that uses variable interpolation could be the following:
```